src/formInputs/DatePicker/index.tsx

```diff
@@ -30,6 +30,7 @@ export interface DatePickerProps {
   readonly alwaysShow: boolean;
   readonly inline?: boolean;
   readonly inputAttributes?: HTMLInputProps & InputProps;
+  readonly showWeekEnd?: boolean;
```

Comment:
```suggestion
  readonly highlightWeekEnd?: boolean;
```
showWeekEnd란 말은 주말을 보여준다라는 의미가 될 수도 있어서 highlightWeekEnd와 같은 이름을 사용해 주시면 좋을 것 같습니다!